admin/controllers/school.js

```diff
@@ -19,9 +20,11 @@ controller.getSchoolLandingPage = async (req, res, next) => {
     const checkWindowData = await checkWindowV2Service.getActiveCheckWindow()
     const schoolName = await schoolService.findSchoolByDfeNumber(req.user.School)
     const featureEligibilityData = schoolHomeFeatureEligibilityPresenter.getPresentationData(checkWindowData, req.user.timezone)
+    const hasIncompleteChecks = await pupilRegisterService.hasIncompleteChecks(req.user.schoolId)
```

Comment:
This is fine but we might want to consider running those await calls in parallel in the future.

Reply:
good spot! that would make a lot of sense.  Is that not an easy win by awaiting a promise.all ?